lib/views/password_generation.dart

```diff
@@ -24,7 +25,7 @@ class _PasswordGenerationPageState extends State<PasswordGenerationPage> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: const Text('Password generation'),
+        title: Text(AppLocalizations.of(context)!.appTitle),
```

Comment:
we should avoid force unwrap wherever possible.  In this case `AppLocalizations.of(context)?.appTitle` should be enough

Reply:
✅